README.md

```diff
@@ -101,6 +101,26 @@ export default AjaxService.extend({
 });
 ```
 
+Headers by default are only passed if the hosts match, or the request is a relitive path.
+You can overwrite this behavior by either passing a host in with the request, setting the
+host for the ajax service, or by setting an array of allowedHosts that can be either
```

Comment:
If you could put `allowedHosts` in backticks, so it's marked as inline code, I'd appreciate it!